fastlane-ios/fastlane/Fastfile

```diff
@@ -0,0 +1,40 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  
+  project_path = "myProject/myProject.xcodeproj"
+
+  desc "Run iOS tests"
+  lane :test do
+    scan(
+      project: project_path
+    )
+  end
+
+  desc "Run SwiftLint"
+  lane :lint do
+    sh "swiftlint"
+  end
+
+  desc "Run build"
+  lane :build do
+    gym(
```

Comment:
Might be too opinionated for here but I've always needed `clean_build_artifacts` & `clear_derived_data` for consistent build outcomes prior to building (even with `clean: true`)